### DISCUSSION TITLE
Support for query params

### DISCUSSION THREAD

**Discussion by josueayala27** [nuxt-community/auth-module#1492] [Ideas & Feature Requests]
A few days ago I wanted to implement an erp that allows me to send configurations or parameters through query parameters, I did not know how to do it.

**Comment by bmulholland** [comment]
Is this a support request ("I did not know how to do it") or a feature request?

**Comment by josueayala27** [comment]
@bmulholland feature request 👻

**Reply by bmulholland** [reply]
Thanks!
